src/views/Registrations/DefineRegistration.vue

```diff
@@ -1,17 +1,100 @@
 <template>
   <div id="define-registration">
-    REGISTRATION PLACEHOLDER
+    <section class="mt-10">
+      <header id="name-info">
+        <h2>1. Name</h2>
+      </header>
+      <v-card flat class="step-container">
+        <NameRequestInfo />
+      </v-card>
+    </section>
+
+    <section class="mt-10">
+      <header id="business-addresses">
+        <h2>2. Business Addresses</h2>
+        <p class="mt-4">Enter the business mailing and delivery addresses. The Delivery Address must be located in
+          British Columbia.
+        </p>
+      </header>
+    </section>
+
+    <section class="mt-10">
+      <header id="business-contact">
+        <h2>3. Business Contact Information</h2>
+        <p class="mt-4">Enter the contact information for the business. The Corporate Registry will use this to
+          communicate with the business in the future, including sending registration documents and notifications.
+        </p>
+      </header>
+      <BusinessContactInfo
+        :initialValue="getBusinessContact"
+        :isEditing="true"
+        :showErrors="null"
+        @contactInfoChange="setBusinessContact($event)"
+        @contactInfoFormValidityChange="onBusinessContactFormValidityChange($event)"
+      />
+    </section>
+
+    <section class="mt-10">
+      <header id="business-start-date">
+        <h2>4. Business Start Date</h2>
+        <p class="mt-4">Enter the start date of the business. The start date can be
+          <v-tooltip top max-width="20rem" content-class="top-tooltip" transition="fade-transition">
+            <template v-slot:activator="{ on }">
+              <span v-on="on" class="tool-tip dotted-underline"> no more than 2 years in the past</span>
```

Comment:
Did you really want the first space char to be underlined?

If needed, you can force a space character elsewhere using `&nbsp;`.

Reply:
Nope, not intentional. Will put the space pre-tooltip. Good catch

Reply:
Updated